### ISSUE TITLE
Add Plex Chinese TC to [What’s new]

### ISSUE THREAD

**Issue by mjabbink** [carbon-design-system/design-language-website#1363]
When IBM Plex Chinese TC is released on GitHub, please plan to have this go up soon after for communication purposes.

`Image reference`
<img width="984" alt="Screenshot 2024-07-12 at 5 01 25 PM" src="https://github.com/user-attachments/assets/56104206-0f20-4581-831b-4a8ddaa9e6f6">

`copy`
After more than two years of design and development, IBM Plex Chinese TC (Traditional) is now available. This will soon be followed by a Plex Chinese SC (Simplified) version, making Plex Chinese a valuable landmark for our users. Plex is open source and free to use. Additionally, Plex is available in Arabic, Cyrillic, Devanagari, Greek, Hebrew, Japanese, Korean, and Thai. You can find more information about the Plex typeface family in the [IBM Design Language](https://www.ibm.com/design/language/typography/typeface) site and download IBM Plex Sans TC from [GitHub](https://github.com/IBM/plex/releases).

**Comment by mjabbink** [comment]
FYI @Lingaraj-13

**Comment by DianaStanciulescu** [comment]
@Lingaraj-13 This looks good, we need an editor to take a look. figma link here: https://www.figma.com/design/A8Aqo87GXAFPJ7gbIvJYkL/%5BIDL%5D-Typeface-%26-What's-new?node-id=603-9791&t=mQ8B8cBdeJGr9sjd-4

**Comment by Lingaraj-13** [comment]
Hello @DianaStanciulescu 
Can we consider the below Figma space for the Devs to refer for the updates once it is marked "Dev ready" ? 
for three IDL pages (Landing page, What's new page and Typeface page) 

https://www.figma.com/design/A8Aqo87GXAFPJ7gbIvJYkL/%5BIDL%5D-Typeface-%26-What's-new?node-id=912-9565&t=315cHUWU95GLGlE9-4

**Comment by DianaStanciulescu** [comment]
@Lingaraj-13 yes - I am actually waiting for a couple more confirmations for 2 text bits. once ready-ready I will let you know. If you want the devs to get a head start , they can use the mockups. any future changes will be minimal. thanks!

**Comment by Lingaraj-13** [comment]
Closing this issue as this ask is completed
